clients/file-client/src/openapi.json

```diff
@@ -1176,6 +1393,9 @@
     }
   },
   "servers": [
+    {
```

Comment:
This is duplicated

Reply:
Weird, it was auto generated this way, going to check